jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -308,8 +313,12 @@ def vue_do_aper_phot(self, *args, **kwargs):
                 ctfac = None
                 sum_ct = None
                 sum_ct_err = None
-
-            if include_flux_scale:
+            if self.telescope == 'JWST':
+                flux = phot_table['sum'][0]
+                flux_scale = flux.to(u.ABmag).value
+                self.flux_scaling = flux_scale
+                sum_mag = -2.5 * np.log10(rawsum.value / flux_scale) * u.mag

```

Comment:
I am very unsure of these calculations. @orifox and @pllim, can you please make sure I'm using the correct variables in the correct way?

Reply:
As discussed offline:

* Hardcode the flux conversion factor from MJy to ABmag when MJy/sr unit is detected in the image metadata.
* Discard changes to hardcode count conversion factor to 1.

Thanks!